pyomo/contrib/fbbt/fbbt.py

```diff
@@ -765,7 +803,7 @@ def visiting_potential_leaf(self, node):
         return False, None
 
 
-def fbbt_con(con, deactivate_satisfied_constraints=False, integer_tol=1e-4):
+def fbbt_con(con, deactivate_satisfied_constraints=False, integer_tol=1e-5, infeasible_tol=1e-8):
```

Comment:
Are fbbt_con / fbbt_block always/only called by fbbt?  If that is the case, would you consider removing the default values for the arguments here as a defensive programming step to guarantee that the caller always remembers to pass in the values from the calling fbbt()?

Reply:
I know that I call `fbbt_block()` from another place.

Reply:
This is a good question/point. I could probably do that and make fbbt_con and fbbt_block private. I'm not sure there is a need for these to be public? @qtothec Is there any reason you could not use ```fbbt()``` instead of ```fbbt_block()```? They do the same thing as long as you are passing them a block and not a constraint. If we do make this change, I will do so in another PR.

Reply:
If they do the same thing when passed a block, then there's no reason, other than that I decided to use the more specific name.

Reply:
I will also put this in another PR.